src/prognostic_equations/advection.jl

```diff
@@ -476,9 +479,11 @@ function edmfx_sgs_vertical_advection_tendency!(
                     ᶜaqʲ,
                     ᶠJ,
                 )
-                @. ᶜqʲₜ += ᶜinv_ρ̂ * vtt
-                @. Yₜ.c.sgsʲs.:($$j).q_tot += ᶜinv_ρ̂ * vtt
-                @. ᶜ∂ρ∂t_sed += vtt
+                vtt_sed_detrainment = @. lazy(min(0,
+                    -1 * ᶜρʲs.:($$j) * ᶜqʲ * ᶜwʲ * ᶜright_biased_∂a∂z))
```

Comment:
Could you add some documentation (either in the docs or in a code comment here) on how this term is derived?